packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -86,7 +98,150 @@ function registerCommands(): vscode.Disposable {
       }
     }
   );
-  return vscode.Disposable.from(customEventHandler);
+  const exceptionBreakpointCmd = vscode.commands.registerCommand(
+    'sfdx.debug.exception.breakpoint',
+    configureExceptionBreakpoint
+  );
+  const startSessionHandler = vscode.debug.onDidStartDebugSession(session => {
+    cachedExceptionBreakpoints.forEach(breakpoint => {
+      const args: SetExceptionBreakpointsArguments = {
+        exceptionInfo: breakpoint
+      };
+      session.customRequest(EXCEPTION_BREAKPOINT_REQUEST, args);
+    });
+  });
+  return vscode.Disposable.from(
+    customEventHandler,
+    exceptionBreakpointCmd,
+    startSessionHandler
+  );
+}
+
+export interface ExceptionBreakpointItem extends vscode.QuickPickItem {
+  typeref: string;
+  breakMode: DebugProtocol.ExceptionBreakMode;
+  uri?: string;
+}
+
+interface BreakModeItem extends vscode.QuickPickItem {
+  breakMode: DebugProtocol.ExceptionBreakMode;
+}
+
+const EXCEPTION_BREAK_MODES: BreakModeItem[] = [
+  {
+    label: nls.localize('always_break_text'),
+    description: '',
+    breakMode: EXCEPTION_BREAKPOINT_BREAK_MODE_ALWAYS
+  },
+  {
+    label: nls.localize('never_break_text'),
+    description: '',
+    breakMode: EXCEPTION_BREAKPOINT_BREAK_MODE_NEVER
+  }
+];
+
+async function configureExceptionBreakpoint(): Promise<void> {
+  const sfdxApex = vscode.extensions.getExtension(
+    'salesforce.salesforcedx-vscode-apex'
+  );
+  if (sfdxApex && sfdxApex.exports) {
+    const exceptionBreakpointInfos: ExceptionBreakpointItem[] = await sfdxApex.exports.getExceptionBreakpointInfo();
```

Comment:
The extension gets a list of System and user-defined exception types from Apex language server.